.golangci.yml

```diff
@@ -13,6 +13,7 @@ linters:
     - goconst
```

Comment:
We will need to add a test to ensure all helm plugin options are set.
I haven't thought of the best way yet.
However, I think we might want to add  e2e tests , similar to the others  https://github.com/kubernetes-sigs/kubebuilder/tree/master/test/e2e ,to mock scenarios with helm and verify that aeverything is as expected.

Reply:
Hi @bavarianbidi 

I thought about this one and IMO: 

1 - We need to add the new flag to the alpha generate
2 - We need to add a new mock in alpha generation that includes the Helm plugin and will utilise this option. Then, at the end, we validate that the helm chart was output in the directory path provided. That should be enough to cover both scenarios in the scope of this PR

Reply:
Please review how the alpha is generated and its associated tests. If you need a hand after that, please feel free to ping me in the Kubebuilder Slack channel. I am happy to help out.